tests/load/script_archive_unused/deploy_with_genlayer_sdk.py

```
@@ -0,0 +1,367 @@
+        import time
+        start_time = time.time()
+        
+        while time.time() - start_time < timeout:
+            request = {
+                "jsonrpc": "2.0",
+                "method": "eth_getTransactionReceipt",
+                "params": [tx_hash],
+                "id": 1
+            }
+            
+            response = requests.post(self.rpc_url, json=request)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)